implentio/ui-toolkit
# Feat/iut 37/sub nav

## Description
This is the subNav items component. This is for navigation items that have multiple subnavigation items. 

## Link to JIRA ticket
https://implentio.atlassian.net/jira/software/projects/IUT/boards/10?selectedIssue=IUT-37

## Applicable Screenshots
![image](https://github.com/implentio/ui-toolkit/assets/93560211/baf42519-1668-4b63-b2be-5746298c02f4)

## Testing Instructions
Outline the steps required to test your changes effectively (Replace the following to what makes sense for your PR):
- Fetch and checkout my branch `git fetch && git checkout <feat/IUT-37/SubNav>`
- npm run storybook
- checkout the stories and change the properties in the story controls and check if it reflects the appropriate behavior

## Checklist:
Check what is applicable by putting an 'x' in between the []:
